README.md

```diff
@@ -176,9 +200,9 @@ A device attribute MUST be one of these:
   </tr>
 </table>
 
-For example, a device with an ID of `686f6d6965` with a `base` and a `windsensor` node would send:
+For example, a device with an ID of `686f6d6965` with a `outdoor-probe` and a `windsensor` node would send:
 
-```
+```cpp
```

Comment:
Indeed the convention was missing a requirement that values are transported as string. I made a proposal to clarify this in #34 .